[Bug] The CI is failing due to error[E0658]: const extern fn definitions are unstable

**Describe the bug**

The CI is failing due to error[E0658]: `const extern fn` definitions are unstable

**To Reproduce**
Steps to reproduce the behavior:
1. Create a repo via the github classroom link
2. Setup it according to the tutorial
3. Trigger CI

**Expected behavior**
The CI can run without trouble

**Screenshots**

Log available at https://github.com/LearningOS/lab0-0-setup-env-run-os1-kxxt/actions/runs/3263696701/jobs/5363158735


**Environment (please complete the following information):**
 - OS: Codespace
 - Rust Version: default version installed by the setup command
 - QEMU Version: default version installed by the setup command

**Additional context**
I have successfully fixed it. I will open a PR soon

--- a/scripts/classroom.yml
+++ b/scripts/classroom.yml
@@ -16,7 +16,7 @@ jobs:
       - uses: actions-rs/toolchain@v1
         with:
           profile: minimal
-          toolchain: nightly-2022-04-11
+          toolchain: nightly-2022-08-05
           components: rust-src, llvm-tools-preview
           target: riscv64gc-unknown-none-elf
       - uses: actions-rs/install@v0.1

--- a/scripts/classroom.yml
+++ b/scripts/classroom.yml
@@ -13,12 +13,6 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
-      - uses: actions-rs/toolchain@v1
-        with:
-          profile: minimal
-          toolchain: nightly-2022-08-05
-          components: rust-src, llvm-tools-preview
-          target: riscv64gc-unknown-none-elf
       - uses: actions-rs/install@v0.1
         with:
           crate: cargo-binutils

